pkg/rancher-components/src/components/RcDropdown/RcDropdown.vue

```diff
@@ -28,15 +28,16 @@ defineProps<{
   ariaLabel?: string
 }>();
 
+const emit = defineEmits(['update:open']);
+
 const {
   isMenuOpen,
   showMenu,
   returnFocus,
   setFocus,
   provideDropdownContext,
   registerDropdownCollection,
-  handleKeydown,
```

Comment:
It looks like `handleKeydown` inadvertently got deleted. This causes a regression where arrow keys can no longer handle selection after opening a dropdown with the mouse. I suspect this got deleted due to a bad resolution for a merge conflict.